packages/nextclade-cli/src/io/http_client.rs

```diff
@@ -61,7 +69,13 @@ impl HttpClient {
 
     let user_agent = format!("{} {}", this_package_name(), this_package_version_str());
 
+    let tls_config = ClientConfig::builder()
+      .dangerous() // …but the rustls_platform_verifier::Verifier is safe
+      .with_custom_certificate_verifier(Arc::new(Verifier::new_with_extra_roots(extra_ca_certs()?)))
+      .with_no_client_auth();
```

Comment:
It does disable client-sent certificates, which I don't think I've seen any of our users need. Browsers _can_ send client certificates, though in most cases that's not used. I don't know if it's strictly necessary for the `rustls-platform-verifiers` crate. It comes from the [example in rustls-platform-verifier's doc](https://docs.rs/rustls-platform-verifier/0.3.4/rustls_platform_verifier/index.html#installation-and-setup), and I noted it was also in the [implementation of `rustls_platform_verifier::tls_config()`](https://docs.rs/rustls-platform-verifier/0.3.4/src/rustls_platform_verifier/lib.rs.html#61-66), which is the "out of the box (no extra certs)" usage of that crate.

Reply:
It's also what reqwest does by default when it constructs the TLS config (i.e. instead of us providing one as we do in this PR):

https://docs.rs/reqwest/latest/src/reqwest/async_impl/client.rs.html#611-615

as we don't construct a reqwest client with an [`identity` field](https://docs.rs/reqwest/latest/reqwest/blocking/struct.ClientBuilder.html#method.identity).